tsdb/mapper.go

```diff
@@ -181,6 +182,11 @@ func (lm *LocalMapper) Open() error {
 		selectTags.add(tsf.selectTags...)
 		whereFields.add(tsf.whereFields...)
 
+		// If we only have tags in our select clause we just return
+		if len(selectFields) == 0 && len(selectTags) > 0 {
```

Comment:
That's a good question.  It feels like maybe this should be an error?  Right now you'll just get an empty result set, but maybe we should say "You need at least one field in your select statement".  That may change in the future of course based on features we add.  Would you prefer an error over an empty result set in this scenario?